### PR TITLE
Update publish.yml for the 1.21 update

### PR DESCRIPTION
**Pull request by ArduFish**
I updated the publish.yml github workflow for Minecraft 1.21.

### CODE REVIEW

**Comment by MagicalAlexey** [comment]
Why not published mod and can you publish 1.20.6 more?

File: coreLib/src/main/java/dev/kosmx/playerAnim/core/data/AnimationFormat.java [hunk 1]
```diff
@@ -11,8 +14,39 @@ public enum AnimationFormat {
     SERVER(null),
     UNKNOWN(null);
 
-    private final String extension;
```
Comment (KosmX) [inline coreLib/src/main/java/dev/kosmx/playerAnim/core/data/AnimationFormat.java:14]: This is not update publish.yml.

**Review by KosmX — CHANGES_REQUESTED** [review]

File: coreLib/src/main/java/dev/kosmx/playerAnim/core/data/AnimationFormat.java (hunk 1 above)
Comment (ArduFish123) [reply coreLib/src/main/java/dev/kosmx/playerAnim/core/data/AnimationFormat.java:14]: I did not change that file

Comment (ArduFish) [reply coreLib/src/main/java/dev/kosmx/playerAnim/core/data/AnimationFormat.java:14]: That's weird

Comment (ArduFish123) [reply coreLib/src/main/java/dev/kosmx/playerAnim/core/data/AnimationFormat.java:14]: Sorry

Comment (ArduFish) [reply coreLib/src/main/java/dev/kosmx/playerAnim/core/data/AnimationFormat.java:14]: I did not change that file

**Pull request closed without merging**